.azure-pipelines/client.test.live.yml

```diff
@@ -46,6 +48,7 @@ jobs:
           PREMIUM_STORAGE_ACCOUNT_KEY: $(java-storage-test-premium-account-key)
           BLOB_STORAGE_ACCOUNT_NAME: $(java-storage-test-blob-account-name)
           BLOB_STORAGE_ACCOUNT_KEY: $(java-storage-test-blob-account-key)
+          AZCONFIG_CONNECTION_STRING: $(java-azconfig-test-connection-string)
```

Comment:
NIT: Consider changing the naming to `APPCONFIG_CONNECTION_STRING` so that it reflects the name of the service.
same for the value